src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -83,16 +83,16 @@ public class DriveTrain extends SubsystemBase {
    * @since 1/30/2023
    */
   public DriveTrain(RobotContainer robotContainer) {
-    this.shuffleboardManager = robotContainer.getShuffleboardManager();
+
+    shuffleboardManager = robotContainer.getShuffleboardManager();
```

Comment:
You can get rid of the "shuffleboardManager" field and remove RobotContainer references from the DriveTrain constructor and import.